README.md

```diff
@@ -1,5 +1,8 @@
 # EventSource [![npm version](http://img.shields.io/npm/v/eventsource.svg?style=flat-square)](http://browsenpm.org/package/eventsource)[![Build Status](http://img.shields.io/travis/EventSource/eventsource/master.svg?style=flat-square)](https://travis-ci.org/EventSource/eventsource)[![NPM Downloads](https://img.shields.io/npm/dm/eventsource.svg?style=flat-square)](http://npm-stat.com/charts.html?package=eventsource&from=2015-09-01)[![Dependencies](https://img.shields.io/david/EventSource/eventsource.svg?style=flat-square)](https://david-dm.org/EventSource/eventsource)
 
+Fork to merge this commit: https://github.com/tigertext/eventsource/commit/e39460e7b852a0b286d7b752efb8f0bcb04f1057#diff-bd6ddbbcfe0037b5a08d70a3b5f4930c
```

Comment:
We don't need the fork message since you're merging back upstream :)

Reply:
Ok, I was not sure of how all of that would work and I wanted to be sure that the first guy tackling this was credited. Also a question,  I needed to update `nyc` to get the coverage working otherwise it would complain about a node folder being missing in the root. Should that be subject to another PR ? Once again sorry for the mistakes it's my first PR so I should have looked better.